src/components/Doc/ComponentDoc.js

```diff
@@ -49,6 +50,22 @@ export const ComponentDoc = ({
         <title>{pageTitle || name}</title>
         <meta name="description" content={description} />
       </Helmet>
+      {stable && (
+        <Box pad={{ bottom: 'medium' }}>
+          <Box
+            alignContent="center"
+            direction="row"
+            background="light-3"
+            pad="xsmall"
+            round="xsmall"
+          >
+            <Text alignSelf="center">
+              <b>New!</b> This component is available on the{' '}
```

Comment:
Playing with some messaging:

"New! This component is in beta and available on the stable branch. Test it out and let us know what you think."

Not sure what the plans are for the content on the /stable page, but assuming it explains what the stable branch is, why it exists (i.e. its purpose), and then how to consume it. If so, it can do the heavy lifting explaining how it is used as a staging ground and new stable components are included in the next release.

Reply:
I like this messaging, updated to say, "New! <component name> is in beta, test it out and let us know what you think. This component is available on the stable branch."

The stable page on the site will have the same info as the stable wiki page with maybe a few tweaks https://github.com/grommet/grommet/wiki/What-is-grommet-stable-and-how-to-use-it%3F